kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -183,6 +185,11 @@ public boolean importMotherRecord(Map<String, Object> record) {
             return false;
         }
 
+        //validate if it's an updated record compared to one from database
+        if (mother.getUpdatedDateNic() != null && (mctsUpdatedDateNic == null || mother.getUpdatedDateNic().isAfter(mctsUpdatedDateNic))) {
+            return true;
```

Comment:
dont you want to return false here because you are dropping the update? Higher up the call stack, this true or false ends up changing the count of records that was accepted/rejected for an mcts pull, right? this behavior is similar to the 3 if checks above right?